docs/spring-graphql-integration.md

```diff
@@ -0,0 +1,119 @@
+## DGS Framework now using Spring GraphQL!
```

Comment:
Also feels very announcement-y. Perhaps we create a new announcements section? And add dates to the page?